app/mailers/application_mailer.rb

```diff
@@ -1,6 +1,11 @@
 # frozen_string_literal: true
 
 class ApplicationMailer < ActionMailer::Base
-  default from: "no-reply@codeforamerica.org"
+  default from: "admin@#{ENV['EMAIL_DOMAIN']}"
   layout "mailer"
+
+  def snap_application_notification(file_name:, recipient_email:)
+    attachments["snap_application.pdf"] = File.read("#{Rails.root}/#{file_name}")
```

Comment:
We are not yet saving it in `/tmp` but I am guessing you will fix that :) 

Will this be problematic on Heroku?